### PR TITLE
[QOLOE-192]Update select input based on design QA.

### PR DESCRIPTION
**Pull request by elvishu**
Minor styling changes on Select input from Design QA:
https://ssq-qol.atlassian.net/browse/QOLOE-192

### CODE REVIEW

File: src/scss/qld-variables.scss
Comment (elvishu) [inline src/scss/qld-variables.scss:1]: Super minor: moved "hint-text" variables together.

**Review by ghazal-salehi — APPROVED** [review]

**Comment by duttonw** [comment]
Please also fix hardcoded selection options in HBS file. This should have already been picked up in uat testing except i don't believe its in use.
https://github.com/qld-gov-au/qgds-bootstrap5/blob/develop/src/components/bs5/select/select.hbs#L31

**Comment by elvishu** [comment]
Sounds good @duttonw.
In order to keep this PR clear and focused, I will fix the hardcoded options in a separate PR under ticket QOLOE-191.
Please approve this PR if you see fit.

**Review by re-becca — APPROVED** [review]
Looks good from visual POV 👍

**Pull request merged**